build.py

```diff
@@ -51,6 +51,11 @@ def sync_folder(src: Path, dst: Path):
     MOD_DLL = MOD_BIN / DLL_NAME
     MOD_PATH = MODS_ROOT / MOD_ID
 
+# check dll has unique name
+if DLL_NAME.lower() == "examplemod.dll":
+    print("Did you forget to rename the DLL in the project settings?")
+    exit(1);
```

Comment:
I think a warning is enough here, maybe link to [https://modding.stacklands.co/en/latest/guides/tutorial.html#step-1-setting-up-the-template](https://modding.stacklands.co/en/latest/guides/tutorial.html#step-1-setting-up-the-template) as well